photo_push_from_o365_to_officespace.ps1

```diff
@@ -1,30 +1,42 @@
 #
 # photo_push_from_o365_to_officespace.ps1 - upload photos from Exchange Online, Azure AD to OfficeSpace
 #
-###############
+###################
+# Script parameters
+###################
+param (
+    [Parameter(Mandatory = $false)]$creds = $false         # (optional) -creds path_to_creds_file
+)                                                          #   If supplied, will override User config for $promptForCreds and $credsFile.
+
+#############
 # User config
-###############
-$azureUsername   = "serviceacct@customer.com"              # user with Azure AD access
+#############
+$promptForCreds  = $true                                   # $true: will prompt user for credentials
+                                                           # $false: will use creds stored in $credsFile
+                                                           # If -creds command line option is provided, this variable is ignored.
 $credsFile       = "C:\ps\serviceacct.creds"               # encrypted user credentials file
+                                                           # If -creds command line option is provided, this variable is ignored.
+$azureUsername   = "serviceacct@customer.com"              # user with Azure AD access
```

Comment:
Why have this defined here at all?  Won't the prompt ask for a username as well as a password?

Reply:
Yes, the prompt would ask for both, but I believe this is used for when $promptForCreds = $false, and would be used further down:
```$azureCredential = New-Object System.Management.Automation.PSCredential -ArgumentList $azureUsername,$azurePassword```

Reply:
So does it need to have the same value is what is inputed at the prompt?